src/spikeinterface/core/baserecording.py

```diff
@@ -811,10 +811,9 @@ def __init__(self, sampling_frequency=None, t_start=None, time_vector=None):
 
     def get_times(self):
         if self.time_vector is not None:
-            if isinstance(self.time_vector, np.ndarray):
-                return self.time_vector
-            else:
-                return np.array(self.time_vector)
+            if not isinstance(self.time_vector, np.ndarray):
```

Comment:
I think you can just always call np.asaray() which by default will just pass the data along if it is already and np.ndarray but will create a copy if it is hdf5, zarr or a memmap.

Reply:
Yes!

Reply:
Then done in last commit :)